doc/usage.md

```diff
@@ -0,0 +1,109 @@
+# CodeCompass usage
+
+## Setup database (one time only)
```

Comment:
I think we should write a using example at this section too:
### Using example:
```bash
mkdir -p /path/to/pgsql_data
initdb -D /path/to/pgsql_data -E "SQL_ASCII"

# Start PostgreSQL server on port 6250
postgres -D /path/to/pgsql_data -p 6250
```